start/sw/web-app/laravel5.md

```diff
@@ -58,10 +58,10 @@ $ composer -V
 $ composer global require "laravel/installer"
 # Place the ~/.config/composer/vendor/bin directory in your PATH so the laravel 
 # executable can be located by your system.
-$ echo 'export PATH="$PATH:$HOME/.config/composer/vendor/bin"' >> ~/.bash_profile
```

Comment:
@mmhyamin I sure already did, but I think the answer you have sent is not the correct one (FTR it's not the accepted one) in this case.
Could you check my proposed solution?
Please clarify the issue you're having if it is not resolved by it (`source !$`, right after the `PATH=". . .`).